esp-hal/src/timer/systimer.rs

```diff
@@ -374,10 +367,16 @@ impl<const CHANNEL: u8> SpecificUnit<'_, CHANNEL> {
     }
 }
 
+const CHANNEL_COUNT: usize = 3;
```

Comment:
3?

Reply:
Thanks - this was a random placeholder that I forgot to fact-check.